### ISSUE TITLE
All entities with SeatEntity in class name are treated as shipyard entities

### ISSUE THREAD

**Issue by blockninja124** [ValkyrienSkies/Valkyrien-Skies-2#1030]
### This issue occurs when only Valkyrien Skies and addons are installed and no other mods

- [X] I have tested this issue and it occurs when no other mods are installed

### Minecraft Version

1.20.x

### Mod Loader

Forge

### Issue description

As you can see from `VSEntityManager.kt` [here](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/blob/1.20.1/main/common/src/main/kotlin/org/valkyrienskies/mod/common/entity/handling/VSEntityManager.kt#L77), the function checks to see if the entities class has 'SeatEntity' in its class name. This is, I assume, to handle VS's built in seat entity. **However** this can cause some really un-expected behaviour with other mods. 

### Issue reproduction

I was trying to debug the rocket entity from Cosmic Horizons for Starlance, and found that it was always in the shipyard. Turns out, its because its class is `RocketSeatEntity`. The only solution I have right now is to mixin the entity manager and remove this behaviour.

Why is this a thing? Theres literally a system for marking entities as shipyard entities with datapack files, so why this hardcoded check? Instead just mark the VS seat entity as a shipyard entity.

**Comment by blockninja124** [comment]
Ok I did some more digging, and found that this is actually intended behaviour. I believe this is an attempt at some basic 'automatic compat' with modded seats. So then I wondered why adding my entity to the `valkyrienskies:world` handler didn't work. Turns out its actually called `valkyrienskies:default`. So I'm closing this issue for now, but I will probably add more info about the handler datapack system on the wiki in the future as it doesn't seem to have a page describing these values.
